--- acinclude.m4~	2004-08-05 09:55:23.000000000 +0000
+++ acinclude.m4	2005-06-21 00:30:38.360001528 +0000
@@ -489,7 +489,7 @@
 	if test x$withval = xyes; then
 
 		AC_MSG_CHECKING([for newer versions of pnmtotiff])
-		if ppmmake 0,0,0 16 16 | pnmtotiff -color -truecolor >/dev/null 2>/dev/null; then
+		if ppmmake 0,0,0 16 16 | pnmtotiff -color -truecolor >pld.test; then
 			AC_MSG_RESULT([found])
 			K3D_PNMTOTIFF_FLAGS="-color -truecolor"
 		else
