diff -urN k3d-0.4.1.0.orig/configure.ac k3d-0.4.1.0.new/configure.ac
--- k3d-0.4.1.0.orig/configure.ac	2004-06-10 10:13:43.000000000 +0200
+++ k3d-0.4.1.0.new/configure.ac	2004-07-06 21:08:19.278511400 +0200
@@ -243,6 +243,7 @@
 
 AC_CONFIG_FILES([
 k3dui/Makefile
+k3dui/application_window.cpp
 k3dui/gtkurl/Makefile
 ])
 
diff -urN k3d-0.4.1.0.orig/docs/user_reference/Makefile.am k3d-0.4.1.0.new/docs/user_reference/Makefile.am
--- k3d-0.4.1.0.orig/docs/user_reference/Makefile.am	2004-06-01 19:41:48.000000000 +0200
+++ k3d-0.4.1.0.new/docs/user_reference/Makefile.am	2004-07-06 21:06:15.562319120 +0200
@@ -9,7 +9,7 @@
  $(wildcard $(top_srcdir)/docs/docbook/k3d/*.css) \
  $(wildcard $(top_srcdir)/docs/docbook/k3d/images/*.png)
  
-install_dir = $(DESTDIR)$(datadir)/doc/k3d/user_reference
+install_dir = $(top_srcdir)/user_reference
 
 all-local: html
 clean-local: html-clean
diff -urN k3d-0.4.1.0.orig/k3dui/application_window.cpp k3d-0.4.1.0.new/k3dui/application_window.cpp
--- k3d-0.4.1.0.orig/k3dui/application_window.cpp	2004-06-01 19:41:49.000000000 +0200
+++ k3d-0.4.1.0.new/k3dui/application_window.cpp	2004-07-06 21:03:40.524888392 +0200
@@ -288,7 +288,7 @@
 	void on_help_manual()
 	{
 		if(application().user_interface())
-			application().user_interface()->browser_navigate((application().share_path() / "../doc/k3d/user_reference/index.html").native_file_string());
+			application().user_interface()->browser_navigate((application().share_path() / "../doc/@PACKAGE@-@VERSION@/user_reference/index.html").native_file_string());
 	}
 
 	/// Called to display an "about" box with program version and copyright information
